packages/imperative/src/cmd/index.ts

```diff
@@ -13,8 +13,8 @@ export * from "./src/doc/profiles/definition/ICommandProfileSchema";
 export * from "./src/doc/profiles/definition/ICommandProfileProperty";
 export * from "./src/doc/profiles/definition/ICommandProfileTypeConfiguration";
 export * from "./src/doc/handler/ICommandHandler";
-export * from "./src/doc/handler/ICommandHandlerResponseChecker";
-export * from "./src/doc/handler/ICommandHandlerResponseValidator";
+// export * from "./src/doc/handler/ICommandHandlerResponseChecker";
+// export * from "./src/doc/handler/ICommandHandlerResponseValidator";
```

Comment:
Quick question: I'm guessing we plan to remove these comments before merging?
Either way, no request for changes here; I just wanted to get an idea of what we plan to do with these line comments.

Reply:
Potentially. I was thinking maybe we would leave them for awhile, and remove them later. After we get a couple releases into V3, and we haven't received any requests from consumers to add any of the removed content back to the exports.

Reply:
We can create an issue with the file paths of all the files that have comments in them. The only reason I didn't want to remove them entirely is because it becomes simpler to restore them if a request does come in.

Reply:
sounds good 😋 
- https://github.com/zowe/zowe-cli/issues/2168